share/rpcauth/rpcauth.py

```diff
@@ -24,6 +25,7 @@ def main():
     parser = ArgumentParser(description='Create login credentials for a JSON-RPC user')
     parser.add_argument('username', help='the username for authentication')
     parser.add_argument('password', help='leave empty to generate a random password or specify "-" to prompt for password', nargs='?')
+    parser.add_argument("-json", help="output to json instead of plain-text", action='store_true')
```

Comment:
The `share/rpcauth/README.md` file should also be updated to explain the argument and capability change in `rpcauth.py`.

nit:
Recommend making the argument be a single letter (e.g. `-j` for json) to follow the existing style with `-h` for help and prevent conflation with `-j -s -o -n`.

Reply:
thank you. yes single "-" makes more sense.

Reply:
ACK for 713ab485d7b0aa86ea5af35e44917ef15a5fb22a

Reply:
Here's an example for the README.md:
```diff
diff --git a/share/rpcauth/README.md b/share/rpcauth/README.md
index 6b5e4446a5..1b3acb1dac 100644
--- a/share/rpcauth/README.md
+++ b/share/rpcauth/README.md
@@ -15,5 +15,5 @@ positional arguments:
 
 optional arguments:
   -h, --help  show this help message and exit
-  -j, -json   output data in json format
+  -j, --json   output data in json format
```